libraries/standard/http/utest/http_utest.c

```diff
@@ -813,7 +813,17 @@ void test_Http_AddRangeHeader_Invalid_Params( void )
     retCode = HTTPClient_AddRangeHeader( &testHeaders,
                                          0 /* rangeStart */,
                                          10 /* rangeEnd */ );
-    TEST_ASSERT_EQUAL( retCode, HTTP_INSUFFICIENT_MEMORY );
+    TEST_ASSERT_EQUAL( HTTP_INSUFFICIENT_MEMORY, retCode );
+
+    /* Length of headers > length of buffer.*/
+    tearDown();
```

Comment:
Explicit call to `teardown` is probably an indication that these should be separate tests. Though this PR is not introducing it, so approving it.